### PR TITLE
Add the chart

### PR DESCRIPTION
**Pull request by pipo02mix**
Towards https://github.com/giantswarm/giantswarm/issues/5264

### CODE REVIEW

File: helm/stormforger-cli-chart/templates/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+kind: ConfigMap
+metadata:
+  name: {{ .Values.name }}
+  labels:
+    app.kubernetes.io/name: {{ .Values.name }}
+    helm.sh/chart: {{ .Chart.Name }}
+    app.kubernetes.io/managed-by: {{ .Values.managed_by }}
+data:
+  test_case.js: |-
+    definition.setTarget("{{ .Values.test.endpoint }}");
```
Comment (rossf7) [inline helm/stormforger-cli-chart/templates/configmap.yaml:11]: <3

File: helm/stormforger-cli-chart/values.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: stormforger
+managed_by: giantswarm
+image:
+  repository: stormforger/cli
+  tag: latest
```
Comment (rossf7) [inline helm/stormforger-cli-chart/values.yaml:5]: I think latest is fine for now we might need to use a tag later.

File: helm/stormforger-cli-chart/values.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+test:
```
Comment (rossf7) [inline helm/stormforger-cli-chart/values.yaml:8]: Nit but can you order the values.

File: helm/stormforger-cli-chart/templates/job.yaml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        app: {{ .Values.name }}
+    spec:
+      restartPolicy: "Never"
+      initContainers:
+      - name: create-test
+        image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+        command: 
+          - "/bin/bash"
+          - "-c"
+          - forge test-case create giantswarm/test cases/test_case.js || forge test-case update giantswarm/test cases/test_case.js
```
Comment (rossf7) [inline helm/stormforger-cli-chart/templates/job.yaml]: So `giantswarm/test` is the name of the test in the Storm Forger UI? Can we template that?

**Review by rossf7 — APPROVED** [review]
LGTM. Just minor stuff no blockers.

File: helm/stormforger-cli-chart/Chart.yaml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for StormForger CLI
+name: stormforger-cli-chart
```
Comment (rossf7) [inline helm/stormforger-cli-chart/Chart.yaml:4]: I need to use the app label to get the pod. Can we just use `stormforger-cli`?

File: helm/stormforger-cli-chart/templates/job.yaml (hunk 4 above)
Comment (pipo02mix) [reply helm/stormforger-cli-chart/templates/job.yaml]: `giantswarm` is the organization and `test` is the name of test case

File: helm/stormforger-cli-chart/Chart.yaml (hunk 5 above)
Comment (pipo02mix) [reply helm/stormforger-cli-chart/Chart.yaml:4]: done

**Comment by pipo02mix** [comment]
merging

**Pull request merged**